lib/ZwaveClient.js

```diff
@@ -1753,7 +1754,15 @@ ZwaveClient.prototype.connect = async function () {
             this.cfg.nodeFilter && this.cfg.nodeFilter.length > 0
               ? this.cfg.nodeFilter.map(n => parseInt(n))
               : undefined
-        }
+        },
+        metrics: this.cfg.disableMetrics
+          ? null
+          : {
+              applicationName:
+                pkgjson.name +
+                (this.cfg.serverEnabled ? '/ zwave-js-server' : ''),
+              applicationVersion: pkgjson.version
+            }
       },
```

Comment:
I've switched to two methods to enable/disable sending usage statistics.

Reply:
https://github.com/zwave-js/node-zwave-js/blob/metrics/docs/api/driver.md#enablestatistics

Reply:
Fixed